### ISSUE TITLE
Checklist

### ISSUE THREAD

**Issue by gbdomubpkm** [jittopjose/DoToDo#13]
Each item in the checklist should be legible in its entirety (portrait or landscape mode). Is it ?
I don't find for
Item 3 in landscape
![screenshot20201003_134623500](https://user-images.githubusercontent.com/50362604/94990983-4bc3ad80-0580-11eb-9b03-96fe429ca7bd.png)
And never in portrait
![screenshot20201003_134611374](https://user-images.githubusercontent.com/50362604/94990986-5120f800-0580-11eb-86cb-8497bec1dead.png)

In my opinion, it should also be possible to modify in writing each element of the list. Is it ?

**Comment by jittopjose** [comment]
I guess we need to allow the text to be in multiline if its longer than the width of the screen. I will do it.

"**In my opinion, it should also be possible to modify in writing each element of the list. Is it ?**"  - I didn't get this point. Could you please explain ?

**Comment by gbdomubpkm** [comment]
Yes when editing, we can delete an item of a checklist (X) but we can't modify the content of the item.
![screenshot20201007_080927156](https://user-images.githubusercontent.com/50362604/95294657-f801de80-0875-11eb-91d2-eb7e44cc469e.png)
Maybe add the modification icon next to the cross for each item.
![screenshot20201007_081229809](https://user-images.githubusercontent.com/50362604/95294703-0e0f9f00-0876-11eb-8eef-99206abaf47c.png)

**Comment by jittopjose** [comment]
@gbdomubpkm  Thanks for the clarification. Now I got your point. I will look into it and will some how bring the edit functionality for each item. I am checking the best way to implement it.
